src/status_im/common/standard_authentication/events.cljs

```diff
@@ -66,9 +66,12 @@
 (defn- bottom-sheet-password-view
   [{:keys [on-press-biometric on-auth-success auth-button-icon-left auth-button-label]}]
   (fn []
-    (let [handle-password-success (fn [password]
-                                    (rf/dispatch [:standard-auth/reset-login-password])
-                                    (-> password security/hash-masked-password on-auth-success))]
+    (let [handle-password-success
+          (fn [password]
+            (let [sha3-pwd (security/hash-masked-password password)]
+              (rf/dispatch [:standard-auth/reset-login-password])
+              (rf/dispatch [:wallet/make-partially-operable-accounts-fully-operable sha3-pwd])
```

Comment:
Isn’t it a costly operation to include this in the authentication process? I’m unsure which is more resource intensive, sending an RPC call for status-go to handle it, or searching through accounts to find partially operable ones?

Reply:
I meant to have a sub which returns a boolean if there are any partially operable key pairs. I have updated the keypair map to have the `lowest-operability` key (while saving it to app-db) which will hold the lowest operable state of accounts generated from it. This greatly reduces the time to go through the accounts data every time. 

On the other hand, I don't mind calling this RPC every time we authenticate even if there are no partially operable key pairs, but we need to ensure the status-go doesn't return any error (just to prevent any surprises). I'm fine with either way.

Reply:
I checked the go code. It's safe to call even if there is no partially operable key pair 👍 

I have a concern. We execute the callback `on-auth-success` along with this dispatch. On the send flow, the transaction is made on auth success. If both RPC calls are made at the same time, most likely, the transaction will fail for partially operable accounts as the key store files may not be created at that time. Should we move the callback (`on-auth-success`) on success and on error of this event? Let me know your thoughts

Reply:
@smohamedjavid please check my last commit, the `lowest-operability` can be ':no' without any `:partially` operable accounts, so i created a new sub to check if there is any `:partially` operable accounts, 

also added operable condition to data-store func to avoid updating `:no` operable in keypair's accounts.

Reply:
Fair enough 👍